WordPress/Classes/ViewRelated/Jetpack/Social/JetpackSocialNoConnectionView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+
+struct JetpackSocialNoConnectionView: View {
+
+    private let viewModel: JetpackSocialNoConnectionViewModel
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 12.0) {
+            HStack(spacing: -6.0) {
+                iconImage("icon-tumblr")
+                iconImage("icon-facebook")
+                iconImage("icon-linkedin")
```

Comment:
Yeah, we will have to make a couple changes to support that. Off the top of my head we'll need:

- Add the service call somewhere so the `PublicizeService` objects are always there. Probably the dashboard? We'll have the no connections view there as well.
- Someway to download the icons. iOS 15 has `AsyncImage` for SwiftUI but we still support iOS 14, so we'll need to create some sort of image view that downloads the icons.

Since these are larger changes, I'll make them in a separate PR.